### ISSUE TITLE
caffe: /usr/bin/java': free(): invalid pointer

### ISSUE THREAD

**Issue by uranium** [bytedeco/javacpp-presets#93]
When doing:
````
            inputLayer.Reshape(numImages, numOfInputLayerChannels, height, width);
            caffeNet.Reshape();
````
from application main method everything works perfect (same holds for TestNG tests).

However, doing something like:

````
final Thread t = new Thread() {

                    public void run() {

                        try {
                            inputLayer.Reshape(numImages, numOfInputLayerChannels, height, width);
                            caffeNet.Reshape();
                        } catch (Exception e) {
                            e.printStackTrace();
                        }
                    }
                };

                t.start();
````
Results in JVM crash and error like: "*** Error in `/usr/bin/java': free(): invalid pointer: 0x0000000801204e00 ***".

Any ideas what could be going on here ?

**Comment by saudet** [comment]
Are you using the latest version from the repository? A lot has changed

**Comment by saudet** [comment]
Check here for some instructions on how to build from source:
https://github.com/bytedeco/javacpp-presets/wiki/Build-Environments

**Comment by saudet** [comment]
Please try again with the newly released version 1.1, thanks!

**Comment by saudet** [comment]
Given the lack of feedback, I'm assuming this is no longer an issue. Please reopen if this is still happening though. Thank you!
